spec/rubocop/cop/style/extra_spacing_spec.rb

```diff
@@ -49,6 +49,28 @@
       expect(cop.offenses).to be_empty
     end
 
+    it 'does not skip single line hash literals with symbol keys' do
+      inspect_source(cop, 'hash = {a:   1,  b:    2}')
+      expect(cop.offenses.size).to eq(3)
+    end
+
+    it 'does not skip single line hash literals with hash rockets' do
+      source = [
+        'let(:single_line_hash) {',
+        '  {"a"   => "1", "b" => "2"}',
+        '}',
+        'let(:multiline_hash) {',
```

Comment:
Good point, I removed the second hash because it wasn't necessary.  I did leave the wrapping `let` block since I wanted the single-line hash to be within a multiline ast.

Updated: 9ce24e4c6cf494853fdd1ce27568744b22219b1f